src/main/java/co/inventorsoft/StreamHomework.java

```diff
@@ -29,7 +36,14 @@
      * @return collection of user, without duplicates
      */
     public List<User> createUsers(final List<String> emails) {
-        return null;
+
+        return emails.stream()
+                .filter(
+                        email -> Optional.ofNullable(email)
+                                .isPresent())
```

Comment:
Hehe, it could have been done simpler: either by checking if the email is not `null` (`email -> email != null`) or using `Objects::nonNull` method reference.
Though, you solution also works fine:)